examples/Nokia_LCD_Demo/fonts/Glyphs_LCD_Fonts.h

```diff
@@ -0,0 +1,294 @@
+        ,{0x08, 0x08, 0x0F, 0x08, 0x08, 0x08} // E0 horizontal line segment with upwards extension
+        ,{0x08, 0x08, 0x0F, 0x00, 0x00, 0x00} // E1 bottom right corner segment
+        ,{0x00, 0x12, 0x1F, 0x10, 0x00, 0x00} // E2 superscript 1
+        ,{0x00, 0x1D, 0x15, 0x17, 0x00, 0x00} // E3 superscript 2
+        ,{0x00, 0x15, 0x15, 0x1F, 0x00, 0x00} // E4 superscript 3
+        ,{0x00, 0x07, 0x04, 0x1F, 0x00, 0x00} // E5 superscript 4
+        ,{0x00, 0x17, 0x15, 0x1D, 0x00, 0x00} // E6 superscript 5
+        ,{0x00, 0x1F, 0x14, 0x1C, 0x00, 0x00} // E7 superscript 6
+        ,{0x00, 0x01, 0x01, 0x1F, 0x00, 0x00} // E8 superscript 7
+        ,{0x00, 0x1F, 0x15, 0x1F, 0x00, 0x00} // E9 superscript 8
+        ,{0x00, 0x07, 0x05, 0x1F, 0x00, 0x00} // EA superscript 9
+        ,{0x00, 0x1F, 0x11, 0x1F, 0x00, 0x00} // EB superscript 0
+        ,{0x04, 0x06, 0xF7, 0x06, 0x04, 0x00} // EC vertical scrollbar up arrow
+        ,{0x20, 0x60, 0xEF, 0x60, 0x20, 0x00} // ED vertical scrollbar down arrow
+        ,{0x03, 0x03, 0xFF, 0x03, 0x03, 0x00} // EE vertical scrollbar indicator, upper position
+        ,{0x18, 0x18, 0xFF, 0x18, 0x18, 0x00} // EF vertical scrollbar indicator, mid position
+        ,{0xC0, 0xC0, 0xFF, 0xC0, 0xC0, 0x00} // F0 vertical scrollbar indicator, lower position
+        ,{0x08, 0x1C, 0x3E, 0x00, 0x08, 0x08} // F1 horizontal scrollbar left arrow
+        ,{0x08, 0x08, 0x00, 0x3E, 0x1C, 0x08} // F2 horizontal scrollbar right arrow
+        ,{0x3E, 0x3E, 0x08, 0x08, 0x08, 0x08} // F3 horizontal scrollbar indicator, left position
+        ,{0x08, 0x08, 0x3E, 0x3E, 0x08, 0x08} // F4 horizontal scrollbar indicator, mid position
+        ,{0x08, 0x08, 0x08, 0x08, 0x3E, 0x3E} // F5 horizontal scrollbar indicator, right position
+        ,{0x3C, 0x24, 0x24, 0x3C, 0x00, 0x00} // F6 checkbox, not selected
+        ,{0x3C, 0x3C, 0x3C, 0x3C, 0x00, 0x00} // F7 checkbox, selected
+        ,{0x7F, 0x00, 0x00, 0x00, 0x7F, 0x00} // F8 double bar
+        ,{0x63, 0x75, 0x79, 0x75, 0x63, 0x00} // F9 hourglass
+        ,{0x08, 0x08, 0x2A, 0x08, 0x08, 0x00} // FA division symbol
+        ,{0x63, 0x55, 0x49, 0x41, 0x41, 0x00} // FB summation symbol
+        ,{0x60, 0x40, 0x7F, 0x01, 0x03, 0x00} // FC integration symbol (single-line)
+        ,{0x00, 0x00, 0xFF, 0x01, 0x03, 0x00} // FD integration symbol top (multi-line)
+        ,{0x60, 0x40, 0x7F, 0x00, 0x00, 0x00} // FE integration symbol bottom (multi-line)
+        ,{0x70, 0x4C, 0x43, 0x4C, 0x70, 0x00} // FF delta
+    };
+
+
+    // Alternatively, it's possible to create a method for getFont instead of lambda
+    //
+    // const unsigned char* getFont(char character)
+    // {
+    //     return kFont_Table[character];
+    // }
+
+    const LcdFont GlyphFont { 
+        [](char c) { return kFont_Table[(uint8_t) c]; }, 
+        kColumns_per_character,
+        NULL,                       // This font has no aditional horizontal spacing
```

Comment:
Please use `nullptr` instead, `NULL` is for `C` :)

Reply:
Done